### PR TITLE
Add landing page

### PR DESCRIPTION
**Pull request by sah-shah7**
-Created Home Page of Quiz App
image

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (TejasShekar) [inline .vscode/settings.json:2]: remove this as its not needed

File: stylesheets/home.css [hunk 2]
```diff
@@ -1,10 +1,50 @@
...
+    font-weight: 800;
+}
+.lead {
+    font-size: 1.2rem;
+    font-weight: 500;
+}
+.quiz-button-primary{
+    color: #11171a;
+    background-color: #1de9b6;
+    border-color: #1de9b6;
```
Comment (haarsh24) [inline stylesheets/home.css:49]: Declare and use these colors

**Review by haarsh24 — APPROVED** [review]
1 minor change .
Rest all looks good , you can merge.

File: stylesheets/home.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    font-family: "Inter UI", "Segoe UI", "Roboto", "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+    font-size: 1rem;
+    font-weight: 400;
+    line-height: 1.5;
+    background-color: var(--black);
+    color: var(--white);
+}
+
+.font-color-white{
+    color:white ;
```
Comment (TejasShekar) [inline stylesheets/home.css:14]: you already have a variable for color white. You can use that.

**Comment by sah-shah7** [comment]
PR approved. Merging.

File: stylesheets/home.css [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+}
+.heading-black {
+    font-weight: 800;
+}
+.lead {
+    font-size: 1.2rem;
+    font-weight: 500;
+}
+.quiz-button-primary{
+    color: #11171a;
```
Comment (TejasShekar) [inline stylesheets/home.css:47]: color variables for these hex values already exist. you can use them here.

**Review by TejasShekar — APPROVED** [review]
suggested a few changes. rest looks good to merge.

**Review by srv-pranjal — COMMENTED** [review]
Good work! Everything looks good to merge.

**Pull request merged**